static/app/views/starfish/components/chart.tsx

```diff
@@ -198,6 +201,45 @@ function Chart({
     });
   }
 
+  let series: Series[] = data.map((values, index) => ({
+    ...values,
+    yAxisIndex: 0,
+    xAxisIndex: 0,
+    id: values.seriesName,
+    color: colors[index],
+  }));
+  let incompleteSeries: Series[] = [];
+
+  const bucketSize =
+    new Date(series[0]?.data[1]?.name).getTime() -
+    new Date(series[0]?.data[0]?.name).getTime();
```

Comment:
Are we guaranteed to have 2 data points here? `new Date(undefined).getTime()` returns `NaN`.

Reply:
Good question. I added a check below to just assume a delay bucket count of `1` if we're not able to get `bucketSize`:

https://github.com/getsentry/sentry/pull/72469/files#diff-c743a95f0d4036d070f8488ec9eda1a53b4c4c8b473d8aff81ad68f0fb9469f1R219

I think it's pretty unlikely this will happen, and we probably short circuit the chart somewhere else in this scenario, but this should work